sdk/identity/identity/src/msal/nodeFlows/msalOpenBrowser.ts

```diff
@@ -174,6 +183,11 @@ export class MsalOpenBrowser extends MsalNode {
     };
 
     const response = await this.publicApp!.getAuthCodeUrl(authCodeUrlParameters);
-    await open(response);
+
+    try {
+      await interactiveBrowserMockable.open(response, { wait: true });
+    } catch (e) {
```

Comment:
2 questions, if you don't mind 😄 

1. I noticed we added `wait: true` to this call, which means we wait for the browser to quit - any concerns with doing that? I'm not too familiar but off the top of my head what happens if instead of a new browser window it opens in a new tab - does that ever happen and does closing the tab do the right thing here? 
2. I noticed we're catching every exception - should we limit to specific errors or specific messages or just always throw a CredentialUnavailable error? I just don't know if anything downstream is handling `CredentialUnavailable` in a special way

Reply:
`wait: true` makes it so we can catch the error. Otherwise the error is thrown but we can't catch it. This call is in a method `openAuthCodeUrl` that doesn't get awaited, so this never gets stuck because of awaiting this open() call. 👍

Reply:
> I noticed we're catching every exception - should we limit to specific errors or specific messages or just always throw a CredentialUnavailable error? I just don't know if anything downstream is handling CredentialUnavailable in a special way

I can't imagine a good reason why `open()` fails that we won't need to handle as a sign that this credential is unavailable. 🙈